### PR TITLE
trigger unchecking all selected items after running a batch action

### PR DESCRIPTION
**Pull request by lfarrell**
https://jira.lib.unc.edu/browse/BXC-2688

### CODE REVIEW

File: static/js/admin/src/action/AjaxCallbackAction.js [hunk 1]
```diff
@@ -125,9 +125,21 @@ define('AjaxCallbackAction', [ 'jquery', 'jquery-ui', 'RemoteStateChangeMonitor'
 					op.context.target.setState("idle");
 				op.complete(data);
 			}
-		}).fail(function(data) {
-			op.alertHandler.alertHandler('error', textStatus + ", " + error);
-			console.error(textStatus, error);
+
+			// Trigger unchecking all checked objects when action completes
+			$(".select_all").trigger('click', function(){
```
Comment (bbpennel) [inline static/js/admin/src/action/AjaxCallbackAction.js:130]: Why does the behavior of the select_all button need to be redefined here? It is already defined here to do the exact same thing:
https://github.com/UNC-Libraries/Carolina-Digital-Repository/blob/fcrepo4/static/js/admin/src/ResultTableView.js#L397

Comment (lfarrell) [reply static/js/admin/src/action/AjaxCallbackAction.js:130]: In the init method it's only bound to the select_all checkbox. So, the user needs to manually click the checkbox for it to run. Maybe we can break it out into a helper?

Comment (bbpennel) [reply static/js/admin/src/action/AjaxCallbackAction.js:130]: Not sure I follow, its being called on the same selector in both cases, `$(".select_all")`. Could we update ResultTableView to cover both cases? It'd be nice to be able to trigger an "unselect all" event/action without having to go through hitting the checkbox, but that may not be practical

I guess this version does have one difference, which is `var checkbox = $(this).children("input").prop("checked", false);` I'm kind of unclear on what that's doing right before toggling the checkbox

Comment (lfarrell) [reply static/js/admin/src/action/AjaxCallbackAction.js:130]: I reduced it a bit. resultObjects and and selectionUpdated only seem to be in scope when wrapped in an event.

**Pull request merged**